packages/app/src/firebaseApp.ts

```diff
@@ -114,11 +114,11 @@ export class FirebaseAppImpl implements FirebaseApp {
         }
 
         return Promise.all(
-          services.map(service => {
-            return service.INTERNAL
-              ? service.INTERNAL.delete()
-              : Promise.resolve();
-          })
+          services
+            .filter(service => 'INTERNAL' in service)
+            .map(service => {
+              return service.INTERNAL!.delete();
```

Comment:
`.map(service => service.INTERNAL!.delete());` should work, no need for the extra `return`.

Reply:
Got it.